lib/geocoder/lookups/olleh.rb

```diff
@@ -188,14 +210,14 @@ def query_url_params(query)
           s = [query.options[:"vx#{x}"], query.options[:"vy#{x}"]]
           hash.merge!({ "VX#{x}" => s[0], "VY#{x}" => s[1]}) unless s[0].nil? && s[1].nil?
         end
-      when "convert_coord"
+      when 'convert_coord'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.